golangci.yml

```diff
@@ -81,7 +82,6 @@ linters:
     - mnd
```

Comment:
Commenting a random line

You can disable gofmt, because gofumpt is also enabled, by enable-all

Reply:
ack, good point (though minor optimization time wise), changing it